sdk/storage/Azure.Storage.Common/src/Errors.cs

```diff
@@ -32,6 +37,14 @@ internal class Errors
         public static InvalidOperationException AccountSasMissingData() =>
             new InvalidOperationException($"Account SAS is missing at least one of these: ExpiryTime, Permissions, Service, or ResourceType");
 
+        public static InvalidOperationException BlobOrContainerMissing(string leaseClient,
+            string blobBaseClient,
+            string blobContainerClient) =>
+            new InvalidOperationException($"{leaseClient} requires either a {blobBaseClient} or {blobContainerClient}");
```

Comment:
This (and anything else only used by Blobs) should move to the https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/src/BlobErrors.cs file.

Nit - we tend to either keep all params on one line or place all of them on their own lines.

Reply:
Updated. Moved Blob related errors to BlobErrors, also made a new line for each parameter that has 3 or more parameters.